Index: External/Tools/FDBuild/Building/AS2/AS2ProjectBuilder.cs
===================================================================
--- External/Tools/FDBuild/Building/AS2/AS2ProjectBuilder.cs	(revision 2725)
+++ External/Tools/FDBuild/Building/AS2/AS2ProjectBuilder.cs	(working copy)
@@ -33,7 +33,7 @@
                 if (File.Exists(mtascPath))
 					return mtascPath;
 				else
-					return "mtasc.exe"; // hope you have it in your environment path!
+					return "mtasc"; // hope you have it in your environment path!
 			}
 		}
 		#endregion
