diff -Naur app.orig/App.as app/App.as
--- app.orig/App.as	2014-01-24 21:11:47.851644200 +0200
+++ app/App.as	2014-01-24 21:25:20.787141400 +0200
@@ -1,5 +1,6 @@
 package 
 {
+   $AppLinks;
    import net.wg.app.IApplication;
    import net.wg.infrastructure.base.meta.IGlobalVarsMgrMeta;
    import net.wg.infrastructure.managers.ISoundManager;
diff -Naur app.orig/net/wg/data/daapi/base/DAAPIDataClass.as app/net/wg/data/daapi/base/DAAPIDataClass.as
--- app.orig/net/wg/data/daapi/base/DAAPIDataClass.as	2014-01-24 21:12:01.403419300 +0200
+++ app/net/wg/data/daapi/base/DAAPIDataClass.as	2014-01-24 21:25:20.798142100 +0200
@@ -160,7 +160,7 @@
          return true;
       }
 
-      function getHash() : Object {
+      public function getHash() : Object {
          return this._hash;
       }
 
diff -Naur app.orig/net/wg/gui/components/advanced/TankIcon.as app/net/wg/gui/components/advanced/TankIcon.as
--- app.orig/net/wg/gui/components/advanced/TankIcon.as	2014-01-24 21:15:21.690875100 +0200
+++ app/net/wg/gui/components/advanced/TankIcon.as	2014-01-24 21:25:20.806142500 +0200
@@ -91,7 +91,7 @@
 
       private var _isPremiumDirty:Boolean = true;
 
-      private const DIRTY_FLAGS:Array;
+      private var DIRTY_FLAGS:Array;
 
       override protected function onDispose() : void {
          this.flag = null;
diff -Naur app.orig/net/wg/gui/components/controls/ScrollingListEx.as app/net/wg/gui/components/controls/ScrollingListEx.as
--- app.orig/net/wg/gui/components/controls/ScrollingListEx.as	2014-01-24 21:12:10.702951200 +0200
+++ app/net/wg/gui/components/controls/ScrollingListEx.as	2014-01-24 21:25:20.816143100 +0200
@@ -444,7 +444,7 @@
          {
             if(this._scrollBarValue  is  Class)
             {
-               _loc1_ = new this._scrollBarValue as Class() as IScrollBar;
+               _loc1_ = new (this._scrollBarValue as Class)() as IScrollBar;
                _loc1_.addEventListener(MouseEvent.MOUSE_WHEEL,this.blockMouseWheel,false,0,true);
                if(_loc1_ != null)
                {
diff -Naur app.orig/net/wg/gui/components/windows/Window.as app/net/wg/gui/components/windows/Window.as
--- app.orig/net/wg/gui/components/windows/Window.as	2014-01-24 21:12:47.458053500 +0200
+++ app/net/wg/gui/components/windows/Window.as	2014-01-24 21:25:20.824143600 +0200
@@ -76,7 +76,7 @@
 
       private const TAB_PADDING_Y:Number = 30;
 
-      private const CONTENT_DEF_PADDING:Padding;
+      private var CONTENT_DEF_PADDING:Padding;
 
       override protected function onDispose() : void {
          super.onDispose();
diff -Naur app.orig/net/wg/gui/lobby/dialogs/SimpleDialog.as app/net/wg/gui/lobby/dialogs/SimpleDialog.as
--- app.orig/net/wg/gui/lobby/dialogs/SimpleDialog.as	2014-01-24 21:12:17.538342200 +0200
+++ app/net/wg/gui/lobby/dialogs/SimpleDialog.as	2014-01-24 21:25:20.832144000 +0200
@@ -35,7 +35,7 @@
 
       private static const TEXT_HEIGHT_PADDING:uint = 3;
 
-      private static const WINDOW_PADDING:Object;
+      private static var WINDOW_PADDING:Object;
 
       private static const LAYOUT_INVALID:String = "layoutInv";
 
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-01-24 21:14:40.114497100 +0200
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-01-24 21:25:20.844144700 +0200
@@ -84,9 +84,9 @@
 
       private var filterDataInvalid:Boolean = false;
 
-      private var nationFilter:DropDownImageText;
+      private var nationFilter:net.wg.gui.components.controls.DropDownImageText;
 
-      private var tankFilter:DropDownImageText;
+      private var tankFilter:net.wg.gui.components.controls.DropDownImageText;
 
       private var checkBoxToMain:CheckBox;
 
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-01-24 21:15:02.176758900 +0200
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-01-24 21:25:20.850145000 +0200
@@ -22,9 +22,9 @@
 
       public static const FILTER_READY:String = "ready";
 
-      public var nationFilter:DropDownImageText;
+      public var nationFilter:net.wg.gui.components.controls.DropDownImageText;
 
-      public var tankFilter:DropDownImageText;
+      public var tankFilter:net.wg.gui.components.controls.DropDownImageText;
 
       public var checkBoxToMain:CheckBox;
 
diff -Naur app.orig/net/wg/gui/lobby/messengerBar/carousel/data/ChannelListItemVO.as app/net/wg/gui/lobby/messengerBar/carousel/data/ChannelListItemVO.as
--- app.orig/net/wg/gui/lobby/messengerBar/carousel/data/ChannelListItemVO.as	2014-01-24 21:13:39.227014500 +0200
+++ app/net/wg/gui/lobby/messengerBar/carousel/data/ChannelListItemVO.as	2014-01-24 21:25:20.855145300 +0200
@@ -10,7 +10,7 @@
          super(param1);
       }
 
-      private static const excluded:Array;
+      private static const excluded:Array = null;
 
       private var _clientID:Number = 0;
 
diff -Naur app.orig/net/wg/gui/lobby/profile/components/ScrollPane.as app/net/wg/gui/lobby/profile/components/ScrollPane.as
--- app.orig/net/wg/gui/lobby/profile/components/ScrollPane.as	2014-01-24 21:13:08.042230800 +0200
+++ app/net/wg/gui/lobby/profile/components/ScrollPane.as	2014-01-24 21:25:20.864145800 +0200
@@ -246,7 +246,7 @@
          {
             if(this._scrollBarValue  is  Class)
             {
-               _loc1_ = new this._scrollBarValue as Class() as IScrollBar;
+               _loc1_ = new (this._scrollBarValue as Class)() as IScrollBar;
                _loc1_.addEventListener(MouseEvent.MOUSE_WHEEL,this.blockMouseWheel,false,0,true);
                if(_loc1_ != null)
                {
diff -Naur app.orig/net/wg/gui/lobby/profile/data/ProfileUserVO.as app/net/wg/gui/lobby/profile/data/ProfileUserVO.as
--- app.orig/net/wg/gui/lobby/profile/data/ProfileUserVO.as	2014-01-24 21:12:18.508397700 +0200
+++ app/net/wg/gui/lobby/profile/data/ProfileUserVO.as	2014-01-24 21:25:20.868146100 +0200
@@ -26,7 +26,7 @@
 
       public var clanPosition:String = "";
 
-      public var clanEmblem;
+      public var clanEmblem:* = null;
    }
 
-}
\ No newline at end of file
+}
diff -Naur app.orig/net/wg/gui/lobby/techtree/constants/IconTextResolver.as app/net/wg/gui/lobby/techtree/constants/IconTextResolver.as
--- app.orig/net/wg/gui/lobby/techtree/constants/IconTextResolver.as	2014-01-24 21:14:49.035007300 +0200
+++ app/net/wg/gui/lobby/techtree/constants/IconTextResolver.as	2014-01-24 21:25:20.872146300 +0200
@@ -12,9 +12,9 @@
          super();
       }
 
-      private static const namedLabels:Vector.<String>;
+      private static var namedLabels:Vector.<String>;
 
-      private static const iconTexts:Vector.<String>;
+      private static var iconTexts:Vector.<String>;
 
       public static function getFromNamedLabel(param1:String) : String {
          var _loc2_:String = Values.EMPTY_STR;
diff -Naur app.orig/net/wg/gui/lobby/techtree/controls/NameAndXpField.as app/net/wg/gui/lobby/techtree/controls/NameAndXpField.as
--- app.orig/net/wg/gui/lobby/techtree/controls/NameAndXpField.as	2014-01-24 21:12:29.340017200 +0200
+++ app/net/wg/gui/lobby/techtree/controls/NameAndXpField.as	2014-01-24 21:25:20.877146600 +0200
@@ -12,7 +12,7 @@
          super();
       }
 
-      private static const statesMap:Object;
+      private static const statesMap:Object = null;
 
       public var xpLabel:String = "earnedXPLabel";
 
diff -Naur app.orig/net/wg/gui/lobby/techtree/data/state/NodeStateCollection.as app/net/wg/gui/lobby/techtree/data/state/NodeStateCollection.as
--- app.orig/net/wg/gui/lobby/techtree/data/state/NodeStateCollection.as	2014-01-24 21:14:52.598211100 +0200
+++ app/net/wg/gui/lobby/techtree/data/state/NodeStateCollection.as	2014-01-24 21:25:20.884147000 +0200
@@ -13,13 +13,13 @@
          super();
       }
 
-      private static const statePrefixes:Array;
+      private static const statePrefixes:Array = null;
 
-      private static const animation:AnimationProperties;
+      private static const animation:AnimationProperties = null;
 
-      private static const nationNodeStates:Vector.<NodeStateItem>;
+      private static const nationNodeStates:Vector.<NodeStateItem> = null;
 
-      private static const itemStates:Vector.<ResearchStateItem>;
+      private static const itemStates:Vector.<ResearchStateItem> = null;
 
       public static function getStateProps(param1:uint, param2:Number, param3:Object) : StateProperties {
          var _loc4_:StateProperties = null;
diff -Naur app.orig/net/wg/gui/login/impl/ErrorStates.as app/net/wg/gui/login/impl/ErrorStates.as
--- app.orig/net/wg/gui/login/impl/ErrorStates.as	2014-01-24 21:16:06.458435600 +0200
+++ app/net/wg/gui/login/impl/ErrorStates.as	2014-01-24 21:25:20.890147300 +0200
@@ -2,20 +2,20 @@
 {
 
 
-   class ErrorStates extends Object
+   public class ErrorStates extends Object
    {
           
       function ErrorStates() {
          super();
       }
 
-      static const NONE:int = 0;
+      public static const NONE:int = 0;
 
-      static const LOGIN_INVALID:int = 1;
+      public static const LOGIN_INVALID:int = 1;
 
-      static const PASSWORD_INVALID:int = 2;
+      public static const PASSWORD_INVALID:int = 2;
 
-      static const LOGIN_PASSWORD_INVALID:int = LOGIN_INVALID | PASSWORD_INVALID;
+      public static const LOGIN_PASSWORD_INVALID:int = LOGIN_INVALID | PASSWORD_INVALID;
    }
 
-}
\ No newline at end of file
+}
diff -Naur app.orig/net/wg/gui/login/impl/LoginPage.as app/net/wg/gui/login/impl/LoginPage.as
--- app.orig/net/wg/gui/login/impl/LoginPage.as	2014-01-24 21:13:15.038631000 +0200
+++ app/net/wg/gui/login/impl/LoginPage.as	2014-01-24 21:25:20.900147900 +0200
@@ -34,7 +34,7 @@
 
       private static const FREE_SPACE_BORDER_FACTOR:Number = 0.18;
 
-      private static const SPARK_ZONE:Rectangle;
+      private static const SPARK_ZONE:Rectangle = null;
 
       private static const SPARK_QUANTITY:uint = 150;
 
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as app/net/wg/gui/prebattle/squad/SquadItemRenderer.as
--- app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-01-24 21:12:51.979312100 +0200
+++ app/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-01-24 21:25:20.908148400 +0200
@@ -120,7 +120,7 @@
             }
             this.setSpeakers(this.model.isPlayerSpeaking,true);
             textField.alpha = this.vehicleNameField.alpha = 1;
-            _loc1_ = this.model.getStateString();
+            var _loc1_:* = this.model.getStateString();
             if(_loc1_ != PrebattleStateString.UNKNOWN)
             {
                this.statusString = _loc1_;
@@ -135,8 +135,8 @@
                   this.showToolTips();
                }
             }
-            _loc2_ = 4.290295975E9;
-            _loc3_ = _loc2_;
+            var _loc2_:* = 4.290295975E9;
+            var _loc3_:* = _loc2_;
             this.updateVoiceWave();
             _loc3_ = this.model.getCurrentColor();
             if(!isNaN(_loc3_))
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadWindow.as app/net/wg/gui/prebattle/squad/SquadWindow.as
--- app.orig/net/wg/gui/prebattle/squad/SquadWindow.as	2014-01-24 21:12:35.492369100 +0200
+++ app/net/wg/gui/prebattle/squad/SquadWindow.as	2014-01-24 21:25:20.916148800 +0200
@@ -64,7 +64,7 @@
          if(param3)
          {
             _loc4_ = new PlayerPrbInfoVO(param3);
-            _loc5_ = 0;
+            var _loc5_:* = 0;
             while(_loc5_ < this.memberList.dataProvider.length)
             {
                _loc6_ = this.memberList.dataProvider.requestItemAt(_loc5_) as PlayerPrbInfoVO;
diff -Naur app.orig/net/wg/infrastructure/managers/IContainerManager.as app/net/wg/infrastructure/managers/IContainerManager.as
--- app.orig/net/wg/infrastructure/managers/IContainerManager.as	2014-01-24 21:11:47.795641000 +0200
+++ app/net/wg/infrastructure/managers/IContainerManager.as	2014-01-24 21:25:20.920149000 +0200
@@ -16,6 +16,8 @@
 
       function isModalViewsExisting() : Boolean;
 
+      function get loader():ILoaderManager;
+
       function set loader(param1:ILoaderManager) : void;
 
       function get lastFocusedView() : IView;
diff -Naur app.orig/scaleform/clik/controls/ButtonGroup.as app/scaleform/clik/controls/ButtonGroup.as
--- app.orig/scaleform/clik/controls/ButtonGroup.as	2014-01-24 21:15:11.720304800 +0200
+++ app/scaleform/clik/controls/ButtonGroup.as	2014-01-24 21:25:20.925149300 +0200
@@ -61,8 +61,8 @@
          for (_loc2_ in this.scope)
          {
             _loc1_ = _loc2_ as DisplayObjectContainer;
-            return _loc1_;
          }
+         return _loc1_;
       }
 
       public function addButton(param1:Button) : void {
diff -Naur app.orig/scaleform/clik/controls/ScrollingList.as app/scaleform/clik/controls/ScrollingList.as
--- app.orig/scaleform/clik/controls/ScrollingList.as	2014-01-24 21:14:47.695930700 +0200
+++ app/scaleform/clik/controls/ScrollingList.as	2014-01-24 21:25:20.935149900 +0200
@@ -396,7 +396,7 @@
          {
             if(this._scrollBarValue  is  Class)
             {
-               _loc1_ = new this._scrollBarValue as Class() as IScrollBar;
+               _loc1_ = new (this._scrollBarValue as Class)() as IScrollBar;
                _loc1_.addEventListener(MouseEvent.MOUSE_WHEEL,this.blockMouseWheel,false,0,true);
                if(_loc1_ != null)
                {
diff -Naur app.orig/scaleform/clik/controls/TextArea.as app/scaleform/clik/controls/TextArea.as
--- app.orig/scaleform/clik/controls/TextArea.as	2014-01-24 21:12:30.346074700 +0200
+++ app/scaleform/clik/controls/TextArea.as	2014-01-24 21:25:20.944150400 +0200
@@ -295,7 +295,7 @@
          {
             if(this._scrollBarValue  is  Class)
             {
-               _loc1_ = new this._scrollBarValue as Class() as IScrollBar;
+               _loc1_ = new (this._scrollBarValue as Class)() as IScrollBar;
                _loc1_.addEventListener(MouseEvent.MOUSE_WHEEL,this.blockMouseWheel,false,0,true);
                if(_loc1_ != null)
                {
diff -Naur app.orig/scaleform/clik/controls/TileList.as app/scaleform/clik/controls/TileList.as
--- app.orig/scaleform/clik/controls/TileList.as	2014-01-24 21:12:15.132204500 +0200
+++ app/scaleform/clik/controls/TileList.as	2014-01-24 21:25:20.954151000 +0200
@@ -442,7 +442,7 @@
          {
             if(this._scrollBarValue  is  Class)
             {
-               _loc1_ = new this._scrollBarValue as Class() as IScrollBar;
+               _loc1_ = new (this._scrollBarValue as Class)() as IScrollBar;
                _loc1_.addEventListener(MouseEvent.MOUSE_WHEEL,this.blockMouseWheel,false,0,true);
                if(_loc1_ != null)
                {
diff -Naur app.orig/scaleform/clik/ui/InputDetails.as app/scaleform/clik/ui/InputDetails.as
--- app.orig/scaleform/clik/ui/InputDetails.as	2014-01-24 21:15:52.826656000 +0200
+++ app/scaleform/clik/ui/InputDetails.as	2014-01-24 21:25:20.958151200 +0200
@@ -21,7 +21,7 @@
 
       public var code:Number;
 
-      public var value;
+      public var value:*;
 
       public var navEquivalent:String;
 
