diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm	2013-10-12 14:57:09.895531900 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm	2013-10-12 15:02:35.611161800 +0300
@@ -30,7 +30,7 @@
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.profile.pages:ProfileSection"), "isActive") type QName(PackageNamespace(""), "Boolean") end
   trait slot QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "_centerOffset") type QName(PackageNamespace(""), "int") value Integer(0) end
   trait slot QName(PackageNamespace(""), "battlesDropdown") type QName(PackageNamespace("net.wg.gui.lobby.profile.components"), "BattlesTypeDropdown") end
-  trait slot QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentData") type QName(PackageNamespace(""), "Object") end
+  trait slot QName(PackageNamespace(""), "currentData") type QName(PackageNamespace(""), "Object") end
   trait slot QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "battlesType") type QName(PackageNamespace(""), "String") end
   trait method QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "draw") flag OVERRIDE
    method
@@ -75,14 +75,14 @@
 
       pop
       getlocal0
-      getproperty         QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentData")
+      getproperty         QName(PackageNamespace(""), "currentData")
       convert_b
 L28:
       iffalse             L33
 
       getlocal0
       getlocal0
-      getproperty         QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentData")
+      getproperty         QName(PackageNamespace(""), "currentData")
       callpropvoid        QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "applyData"), 1
 
 L33:
@@ -504,7 +504,7 @@
 
       getlocal0
       getlocal2
-      initproperty        QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentData")
+      initproperty        QName(PackageNamespace(""), "currentData")
 
       getlocal0
       getlocal1
