diff -Naur Application.orig-0.orig/net/wg/app/impl/Application.class.asasm Application.orig-0/net/wg/app/impl/Application.class.asasm
--- Application.orig-0.orig/net/wg/app/impl/Application.class.asasm	2013-10-03 11:10:17.373971000 +0300
+++ Application.orig-0/net/wg/app/impl/Application.class.asasm	2013-10-03 11:16:56.683352100 +0300
@@ -28,7 +28,8 @@
      getglobalscope
      pushstring          "windows.swf"
      pushstring          "toolTips.swf"
-     newarray            2
+     pushstring          "../../../xvm/mods/xvm.swf"
+     newarray            3
      call                1
      initproperty        QName(PrivateNamespace(null, "net.wg.app.impl:Application/instance#0"), "_librariesList")
 
diff -Naur Application.orig-0.orig/net/wg/gui/components/tooltips/ToolTipTankmen.class.asasm Application.orig-0/net/wg/gui/components/tooltips/ToolTipTankmen.class.asasm
--- Application.orig-0.orig/net/wg/gui/components/tooltips/ToolTipTankmen.class.asasm	2013-10-03 11:10:19.223971000 +0300
+++ Application.orig-0/net/wg/gui/components/tooltips/ToolTipTankmen.class.asasm	2013-10-03 11:16:56.693352200 +0300
@@ -427,20 +427,24 @@
 
       getlocal0
       getproperty         QName(PackageNamespace(""), "vehicleIco")
-      getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "topPosition")
-      pushbyte            35
-      add
-      setproperty         QName(PackageNamespace(""), "y")
-
-      getlocal0
-      getproperty         QName(PackageNamespace(""), "vehicleIco")
+      getlex              QName(PackageNamespace(""), "Math")
       getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "contentMargin")
       getproperty         QName(PackageNamespace(""), "left")
       getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "bgShadowMargin")
       getproperty         QName(PackageNamespace(""), "left")
       add
+      callproperty        QName(PackageNamespace(""), "round"), 1
       setproperty         QName(PackageNamespace(""), "x")
 
+      getlocal0
+      getproperty         QName(PackageNamespace(""), "vehicleIco")
+      getlex              QName(PackageNamespace(""), "Math")
+      getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "topPosition")
+      pushdouble          41
+      add
+      callproperty        QName(PackageNamespace(""), "round"), 1
+      setproperty         QName(PackageNamespace(""), "y")
+
       getlocal            5
       getlex              QName(PackageNamespace(""), "MENU")
       getlocal0
@@ -519,17 +523,17 @@
       getproperty         QName(PackageNamespace(""), "tankInfoTF")
       getlocal0
       getproperty         QName(PackageNamespace(""), "vehicleIco")
-      getproperty         QName(PackageNamespace(""), "y")
-      pushbyte            1
-      add
-      setproperty         QName(PackageNamespace(""), "y")
+      getproperty         QName(PackageNamespace(""), "x")
+      setproperty         QName(PackageNamespace(""), "x")
 
       getlocal0
       getproperty         QName(PackageNamespace(""), "tankInfoTF")
-      getlocal0
-      getproperty         QName(PackageNamespace(""), "vehicleIco")
-      getproperty         QName(PackageNamespace(""), "x")
-      setproperty         QName(PackageNamespace(""), "x")
+      getlex              QName(PackageNamespace(""), "Math")
+      getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "topPosition")
+      pushdouble          37
+      add
+      callproperty        QName(PackageNamespace(""), "round"), 1
+      setproperty         QName(PackageNamespace(""), "y")
 
       getlocal0
       getproperty         QName(PackageNamespace(""), "vehicleIco")
@@ -1335,30 +1339,18 @@
 
 L15:
       getlocal0
-      getproperty         QName(PackageNamespace(""), "vehicleIco")
-      pushbyte            255
-      setproperty         QName(PackageNamespace(""), "scaleX")
-
+      getproperty         QName(PackageNamespace(""), "tankInfoTF")
+      getlex              QName(PackageNamespace(""), "Math")
       getlocal0
       getproperty         QName(PackageNamespace(""), "vehicleIco")
-      getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "contentMargin")
-      getproperty         QName(PackageNamespace(""), "left")
-      getlex              QName(ProtectedNamespace("net.wg.gui.components.tooltips:ToolTipBase"), "bgShadowMargin")
-      getproperty         QName(PackageNamespace(""), "left")
-      add
+      getproperty         QName(PackageNamespace(""), "x")
       getlocal0
       getproperty         QName(PackageNamespace(""), "vehicleIco")
       getproperty         QName(PackageNamespace(""), "width")
       add
-      setproperty         QName(PackageNamespace(""), "x")
-
-      getlocal0
-      getproperty         QName(PackageNamespace(""), "tankInfoTF")
-      getlocal0
-      getproperty         QName(PackageNamespace(""), "vehicleIco")
-      getproperty         QName(PackageNamespace(""), "x")
       pushbyte            5
       add
+      callproperty        QName(PackageNamespace(""), "round"), 1
       setproperty         QName(PackageNamespace(""), "x")
 
       returnvoid
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileAchievementsSection.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileAchievementsSection.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileAchievementsSection.class.asasm	2013-10-03 11:10:18.803971000 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileAchievementsSection.class.asasm	2013-10-03 11:19:00.104824300 +0300
@@ -90,10 +90,10 @@
       initproperty        QName(PrivateNamespace(null, "net.wg.gui.lobby.profile.pages:ProfileAchievementsSection/instance"), "isRareAchievementRequested")
 
       findpropstrict      QName(PackageNamespace(""), "requestDataS")
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       iffalse             L26
 
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       getproperty         QName(PackageNamespace(""), "id")
       coerce              QName(PackageNamespace(""), "Object")
       jump                L28
@@ -114,13 +114,13 @@
       iffalse             L39
 
       pop
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       convert_b
 L39:
       iffalse             L43
 
       findpropstrict      QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "updateByDossier")
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       callpropvoid        QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "updateByDossier"), 1
 
 L43:
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm	2013-10-03 11:10:18.803971000 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/ProfileSection.class.asasm	2013-10-03 11:18:17.894164200 +0300
@@ -29,7 +29,7 @@
   trait slot QName(PackageNamespace(""), "layoutManager") type QName(PackageNamespace("net.wg.gui.lobby.profile.data"), "SectionLayoutManager") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.profile.pages:ProfileSection"), "isActive") type QName(PackageNamespace(""), "Boolean") end
   trait slot QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "_centerOffset") type QName(PackageNamespace(""), "int") value Integer(0) end
-  trait slot QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier") type QName(PackageNamespace("net.wg.data.gui_items.dossier"), "AccountDossier") end
+  trait slot QName(PackageNamespace(""), "currentDossier") type QName(PackageNamespace("net.wg.data.gui_items.dossier"), "AccountDossier") end
   trait method QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "draw") flag OVERRIDE
    method
     refid "net.wg.gui.lobby.profile.pages:ProfileSection/instance/net.wg.gui.lobby.profile.pages:ProfileSection:draw"
@@ -292,13 +292,13 @@
       setlocal2
 
       getlocal0
-      getproperty         QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getproperty         QName(PackageNamespace(""), "currentDossier")
       getlocal2
       ifeq                L17
 
       getlocal0
       getlocal2
-      initproperty        QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      initproperty        QName(PackageNamespace(""), "currentDossier")
 
       getlocal0
       getlocal2
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/awards/ProfileAwards.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/awards/ProfileAwards.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/awards/ProfileAwards.class.asasm	2013-10-03 11:10:21.823971000 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/awards/ProfileAwards.class.asasm	2013-10-03 11:19:09.975021700 +0300
@@ -270,10 +270,10 @@
 
       getlocal0
       findpropstrict      QName(PackageNamespace("net.wg.data.gui_items.dossier"), "AccountDossier")
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       iffalse             L23
 
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       getproperty         QName(PackageNamespace(""), "id")
       coerce_s
       jump                L25
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/technique/ProfileTechnique.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/technique/ProfileTechnique.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/technique/ProfileTechnique.class.asasm	2013-10-03 11:10:22.633971000 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/technique/ProfileTechnique.class.asasm	2013-10-03 11:19:15.905140300 +0300
@@ -130,7 +130,7 @@
 
       findpropstrict      QName(PackageNamespace(""), "requestDataS")
       pushstring          "userName"
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       getproperty         QName(PackageNamespace(""), "id")
       pushstring          "vehicleId"
       getlocal2
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/pages/technique/ProfileTechniquePage.class.asasm Application.orig-0/net/wg/gui/lobby/profile/pages/technique/ProfileTechniquePage.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/pages/technique/ProfileTechniquePage.class.asasm	2013-10-03 11:10:22.643971000 +0300
+++ Application.orig-0/net/wg/gui/lobby/profile/pages/technique/ProfileTechniquePage.class.asasm	2013-10-03 11:19:24.005302300 +0300
@@ -139,11 +139,11 @@
       getlocal0
       pushscope
 
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       iffalse             L7
 
       getlocal0
-      getlex              QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages:ProfileSection"), "currentDossier")
+      getlex              QName(PackageNamespace(""), "currentDossier")
       callpropvoid        QName(ProtectedNamespace("net.wg.gui.lobby.profile.pages.technique:ProfileTechniquePage"), "updateByDossier"), 1
 
 L7:
